test/testdata/check.golden

```diff
@@ -1,9 +1,12 @@
 kubernetes-api: can initialize the client..................................[ok]
 kubernetes-api: can query the Kubernetes API...............................[ok]
 kubernetes-api: is running the minimum Kubernetes API version..............[ok]
-linkerd-api[namespace]: control plane namespace exists.....................[ok]
+linkerd-api: control plane namespace exists................................[ok]
+linkerd-api: can initialize the client.....................................[ok]
+linkerd-api: can query the control plane API...............................[ok]
 linkerd-api[kubernetes]: control plane can talk to Kubernetes..............[ok]
 linkerd-api[prometheus]: control plane can talk to Prometheus..............[ok]
+linkerd-version: can get the latest version................................[ok]
```

Comment:
slight naming nit: can we use "can query for the latest version" or "can determine the latest version" instead? "can get" to me implies some downloading (I wasn't sure what it meant at first)

Reply:
Ah, yep, great call! I like "can determine the latest version" a lot -- will update.